src/drivers/gps/params.c

```diff
@@ -122,12 +122,30 @@ PARAM_DEFINE_INT32(GPS_UBX_MODE, 0);
  * @max 360
  * @unit deg
  * @reboot_required true
- * @decimal 0
+ * @decimal 3
```

Comment:
Can you really measure this with millidegree accuracy?

Reply:
Individual measurements may only be in 0.1~0.01deg accuracy. An offline calibration procedure can recover sub 0.01deg offsets. So millidegree is a good resolution for this parameter.